cdap-common/src/main/java/io/cdap/cdap/common/lang/PackageFilterClassLoader.java

```diff
@@ -129,4 +130,9 @@ private String getResourcePackage(String classResource) {
     }
     return packageName;
   }
+
+  @Override
+  public void close() throws IOException {
+    this.bootstrapClassLoader.close();
```

Comment:
We shouldn't close the bootstrapclassloader. This is for loading JVM classes, which shouldn't be closed and shared among all classloaders.

Reply:
This is not the real bootstrap class loader. Its a regular URL class loader with null parent. Its created on line 48 in this file.